test-lib.sh

```diff
@@ -208,6 +208,46 @@ function ct_doc_content_old() {
 }
 
 
+# ct_mount_ca_file
+# ------------------
+# Check if /etc/pki/certs/RH-IT-Root-CA.crt file exists
+# return mount string for containers
+ct_mount_ca_file()
+{
+    local mount_parameter=""
+    local ca_full_path="/etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt"
+    if [ -e "$ca_full_path" ]; then
+        mount_parameter="-v $ca_full_path:$ca_full_path:z"
+    fi
+    echo "$mount_parameter"
+}
+
+# ct_build_s2i_npm_variables URL_TO_NPM_JS_SERVER
+# ------------------------------
+# Function returns -e NPM_MIRROR and -v MOUNT_POINT_FOR_CAFILE
+function ct_build_s2i_npm_variables() {
+    local npm_host=$1
+    local ca_full_path="/etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt"
+    set +e
+    npm_mirror=$(echo $npm_host | cut -d'/' -f3)
+    if [ x"$npm_mirror" != "" ]; then
+        # npm_host is in format
+        # http{s}://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org
+        ping -c 5 $npm_mirror > /dev/null
+        if [ $? -ne 0 ]; then
```

Comment:
You can do `if ping -c 5 "$npm_mirror" > /dev/null; then` instead. That way you can even remove the `set +e` above

Reply:
This is not needed at all. We will have $NPM_REGISTRY variable defined. For CI systems.
Therefore I don't see any reason to have this check.